selectiveInference/R/funs.fixed.R

```diff
@@ -420,11 +452,15 @@ debiasingRow = function (Xinfo,               # could be X or t(X) %*% X / n dep
                             nactive, 
                             kkt_tol, 
                             objective_tol, 
-                            max_active) 
+			    parameter_tol,
+                            max_active,
+			    objective_stop,
+			    kkt_stop,
+			    parameter_stop)
       } else {
-          Xsoln = rep(0, nrow(Xinfo))
```

Comment:
This was the bug -- should remain as X %*% soln but it was reinitialized each time